DESCRIPTION

```diff
@@ -28,7 +28,8 @@ Imports:
     tibble,
     tidyr,
     tools,
-    xtable
+    xtable,
+    r2rtf
```

Comment:
Maybe this is not too obvious but the dependencies should be listed in alphabetical orders.